regression/cbmc-java/Makefile

```diff
@@ -18,3 +18,17 @@ show:
 			vim -o "$$dir/*.java" "$$dir/*.out"; \
 		fi; \
 	done;
+
+
+%.class: %.java ../../src/org.cprover.jar
```

Comment:
We currently commit the class files 
I'm generally in favour of generating them instead of putting them into the repo. This would allow us to test different jdk versions. However, this causes lots of other complications, such as some tests that target specific jdk versions only or tests that requre bytecodes that we don't know how to generate with any current Java compiler. Hence, the need for class files in the repo won't disappear completely. Anyways, this would concern all Java regression tests and can't just be done for a single directory.